clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -111,6 +111,11 @@ class Util {
   /// \param Tmpl  whether the class is template instantiation or simple record
   static bool isSyclType(QualType Ty, StringRef Name, bool Tmpl = false);
 
+  /// Checks whether given clang type is a standard SYCL API accessor class.
+  /// \param Ty    the clang type being checked
+  /// \param Tmpl  whether the class is template instantiation or simple record
+  static bool isSyclAccessorType(QualType Ty, bool Tmpl = false);
```

Comment:
Sorry I didn't notice this earlier. Is the second argument required? As far as I can see, you pass true for all cases.

Reply:
The second argument is not needed. I have removed it.